spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "Foods Index Page" do
+  context "user submits 'sweet potato' as query in serach form" do
```

Comment:
Again, it's best to use webmock/VCR to ensure that we stub the external api consumption.